wp-js-plugin-starter.php

```diff
@@ -31,7 +31,9 @@ function wp_js_plugin_starter_register_block() {
 	wp_register_script(
 		'wp-js-plugin-starter',
 		wp_js_plugin_starter_url( 'dist/index.js' ),
-		array( 'wp-element' )
+		array( 'wp-element' ),
+		'1.0.1',
+		true
```

Comment:
I don't think that's the way to fix the issue. The real issue is that if you use `wp.editor` for instance, you need to add `wp-editor` as a dependency etc...